src/js/modules/helpers.js

```diff
@@ -0,0 +1,15 @@
+export function loadAllSliderImages(sectionClass) {
+
+	const observer = new IntersectionObserver(([entry], observer) => {
+
+		if (entry.isIntersecting) {
```

Comment:
you start loading images when section with slider is already partially in viewport. Make it little bit beforehand. For example 150-200px.

Reply:
Done